### ISSUE TITLE
Try-To-Apply: The APPLYING for Food Stamps challenge

### ISSUE THREAD

**Issue by lippytak** [codeforamerica/project-ideas#15]
(Placeholder for now. Someone please expand!)

"The food stamp challenge is one of the best ways to learn about the nutrition hurdles that low-income Americans face every day. The Challenge usually entails eating on a budget of roughly $1 to $1.25 per meal (per person), an amount that approximates the average allocation of food stamp benefits."
via http://www.foodstamped.com/get-involved/take-the-challenge

Eating healthy on a food stamps budget is only the last hurdle of many. The first hurdle is getting enrolled. Let's build empathy/understanding around that by running the APPLY for Food Stamps challenge (with a spiffier) name. NDOC opportunity? @dget raised the good point that we would need to figure out a way not to clog the system with bad applications...possibly in partnerships with HSAs, setting up a dummy site, etc.

**Comment by daguar** [comment]
DAMNIT Jake, I was just adding this! :package: 

I'm editing because I like the name "Try To Apply" -- it rhymes!

JSS: Solid edit. Alliteration is critical for community building. BTW I can edit your GH comments??

PS. Let's go.

PSS. Bring it.

On Wed, May 7, 2014 at 2:39 PM, Jake Solomon notifications@github.comwrote:

> (Placeholder for now. Someone please expand!)
> 
> "The food stamp challenge is one of the best ways to learn about the
> nutrition hurdles that low-income Americans face every day. The Challenge
> usually entails eating on a budget of roughly $1 to $1.25 per meal (per
> person), an amount that approximates the average allocation of food stamp
> benefits."
> - via http://www.foodstamped.com/get-involved/take-the-challenge
> 
> Eating healthy on a food stamps budget is only the last hurdle of many.
> The first hurdle is getting enrolled. Let's build empathy/understanding
> around that by running the APPLY for Food Stamps challenge (with a
> spiffier) name. NDOC opportunity? @dget https://github.com/dget raised
> the good point that we would need to figure out a way not to clog the
> system with bad applications...possibly in partnerships with HSAs, setting
> up a dummy site, etc.
> 
> —
> Reply to this email directly or view it on GitHubhttps://github.com/codeforamerica/health-project-ideas/issues/15
> .

## 

Dave Guarino
Consultant, Health Vertical
(2013 Fellowship alumnus)
Code for America http://www.codeforamerica.org/
dave@codeforamerica.org
LinkedIn http://www.linkedin.com/in/daveguarino/ |
GitHubhttps://github.com/daguar/

**Comment by jimmmyc** [comment]
I think this is an interesting idea. I went through the application process this afternoon and, as expected, it's pretty clunky. There's a circuitous online application process (took me about 30 minutes) that resulted in being approved to have an in-person interview, which appears to be where the actual accept/reject decisions are made.

Thoughts on how this "try to apply" idea would work? Particular reasons to favor food stamps rather than other government services like child care, medicaid, unemployment, etc?

**Comment by daguar** [comment]
> Thoughts on how this "try to apply" idea would work? 

I think, ultimately, this is likely more similar to advocacy work in the sense that it is about identifying the key decisionmakers/overseers with the power to make changes to these programs, and then figure out if there's a way to get them to publicly commit to going through the process.

That said, @lippytak has been working for a while on ["Citizen Onboard"[(http://codeforamerica.github.io/citizen-onboard/#1), which is essentially an annotated walkthrough presentation of what these application processes are really like.

So if you're interested in contributing to this idea more generally, I'd say doing one of those for your own program of interest would be a great first step.

> Particular reasons to favor food stamps rather than other government services like child care, medicaid, unemployment, etc?

I think it'd be great to do this for LOTS of programs!

**Comment by jimmmyc** [comment]
I agree that building empathy by asking folks to go through the application process is a good idea. Not to hijack the thread, but what are some projects that go beyond that, and exist as alternative interfaces for government services? TurboTax for X (where X = SNAP, food medicaid, immigration, etc).

**Comment by lippytak** [comment]
There are some but not enough. A few that come to mind:
- There numerous for FAFSA apps, like http://www.fafsa-application.com
- There are many for getting other types of student loans
- TurboTax is one of many for state/federal taxes
- http://opencounter.us/ is attempting to do this for business permitting at the city level
- Do you know of others?

As a general rule I don't think we should expect to see high quality alternate interfaces for gov services that primarily serve low income groups (SNAP, Medicaid, etc.) because there isn't a profit opportunity. The recent Farm Bill actually bans per-application bounties (referral bonuses ) so it destroys the already weak business model for doing application support. This is why Try-To-Apply is especially important for services for low income groups; fixing the gov service is more important when the market won't create alternatives.

**Comment by daguar** [comment]
Still a good idea.
